examples/wearable/UIComponents/contents/navigationelements/assist-panel.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+	<title>
+		Assist Panel
+	</title>
+	<link href="../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../lib/tau/wearable/theme/default/tau.circle.css"rel="stylesheet" />
+	<link href="../../css/style.css" rel="stylesheet" />
+	<link href="../../css/style.circle.css"rel="stylesheet" />
+	<script src="../../js/circle-helper.js">
+	</script>
+	<script src="../../lib/tau/wearable/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="assist-panel-page">
+		<header class="ui-header">
+			<h2 class="ui-title">
+				Assist Panel
+			</h2>
+		</header>
+		<div class="ui-content">
+			Empty content
+		</div>
+		<div class="ui-assist-panel" id="assistPanel" data-drag-edge="50" data-position="down">
```

Comment:
as we discussed data-position should be default for down

Reply:
Done. Removed.